rollup.config.js

```diff
@@ -0,0 +1,43 @@
+import resolve from "@rollup/plugin-node-resolve";
+import commonjs from "@rollup/plugin-commonjs";
+import sourceMaps from "rollup-plugin-sourcemaps";
+import { babel } from "@rollup/plugin-babel";
+import json from "@rollup/plugin-json";
+import replace from "@rollup/plugin-replace";
+import typescript from "rollup-plugin-typescript2";
+
+import pkg from "./package.json";
+
+const extensions = [".ts"];
+
+/**
+ * @type {import('rollup').RollupOptions}
+ */
+const config = {
+    input: "./src/index.ts",
+    output: [
+        { file: pkg.main, format: "cjs", sourcemap: true },
+        { file: pkg.module, format: "es", sourcemap: true },
+    ],
+    external: ["tslib", "core-js", "keyborg"],
```

Comment:
does core-js need to be there ? as we said we shouldn't use before we kill IE11 support

```suggestion
    external: ["tslib", "keyborg"],
```